app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/credit_card/AdyenCreditCardScreen.kt

```diff
@@ -0,0 +1,418 @@
+        onContactUsClick = onContactUs,
+      )
+
+      else -> PortraitPaymentErrorView(
+        message = message,
+        description = description,
+        onRetryClick = onRetryClick,
+        onContactUsClick = onContactUs,
+      )
+    }
+  }
+}
+
+@Composable
+private fun AdyenCreditCardScreen(
+  modifier: Modifier = Modifier,
+  packageName: String,
+  onBuyClickEnabled: Boolean,
+  onBuyClick: () -> Unit,
+  onOtherPaymentMethodsClick: () -> Unit,
+  paymentView: @Composable () -> Unit,
+) {
+  val configuration = LocalConfiguration.current
+  when (configuration.orientation) {
+    Configuration.ORIENTATION_LANDSCAPE -> {
+      AdyenCreditCardScreenLandscape(
+        modifier = modifier,
+        packageName = packageName,
+        onBuyClickEnabled = onBuyClickEnabled,
+        onBuyClick = onBuyClick,
+        onOtherPaymentMethodsClick = onOtherPaymentMethodsClick,
+        paymentView = paymentView
+      )
+    }
+
+    else -> {
+      AdyenCreditCardScreenPortrait(
+        modifier = modifier,
+        packageName = packageName,
+        onBuyClickEnabled = onBuyClickEnabled,
+        onBuyClick = onBuyClick,
+        onOtherPaymentMethodsClick = onOtherPaymentMethodsClick,
+        paymentView = paymentView
+      )
+    }
+  }
+}
+
+@Composable
+private fun AdyenCreditCardScreenLandscape(
+  modifier: Modifier = Modifier,
+  packageName: String,
+  onBuyClickEnabled: Boolean,
+  onBuyClick: () -> Unit,
+  onOtherPaymentMethodsClick: () -> Unit,
+  paymentView: @Composable () -> Unit,
+) {
+  Column(
+    modifier = modifier
+      .fillMaxSize()
+      .verticalScroll(rememberScrollState())
+      .padding(horizontal = 16.dp)
+      .padding(top = 16.dp, bottom = 8.dp)
+  ) {
+    Row {
+      PurchaseInfoRow(
+        modifier = Modifier.fillMaxWidth(0.4f),
+        buyingPackage = packageName,
+      )
+      paymentView()
+    }
+    PaymentButtons(
+      modifier = Modifier.padding(top = 16.dp),
+      onBuyClickEnabled = onBuyClickEnabled,
+      onBuyClick = onBuyClick,
+      onOtherPaymentMethodsClick = onOtherPaymentMethodsClick
+    )
+  }
+}
+
+@Composable
+private fun AdyenCreditCardScreenPortrait(
+  modifier: Modifier = Modifier,
+  packageName: String,
+  onBuyClickEnabled: Boolean,
+  onBuyClick: () -> Unit,
+  onOtherPaymentMethodsClick: () -> Unit,
+  paymentView: @Composable () -> Unit,
+) {
+  Column(
+    modifier = modifier
+      .padding(horizontal = 16.dp)
+      .padding(top = 16.dp, bottom = 8.dp),
+  ) {
+    PurchaseInfoRow(buyingPackage = packageName)
+    paymentView()
+    PaymentButtons(
+      modifier = Modifier.padding(top = 16.dp),
+      onBuyClickEnabled = onBuyClickEnabled,
+      onBuyClick = onBuyClick,
+      onOtherPaymentMethodsClick = onOtherPaymentMethodsClick
+    )
+  }
+}
+
+@Composable
+private fun AdyenCreditCardView(
+  modifier: Modifier = Modifier,
+  cardComponent: CardComponent,
+) {
+  val lifecycleOwner = LocalLifecycleOwner.current
+  AndroidView(
```

Comment:
Fields text is currently white and not visible...

Reply:
The styles for these fields were added